### PR TITLE
Added story line: usage example in README

### PR DESCRIPTION
**Pull request by eriktks**
Ref: #12 

Added usage example in README which can function as a basic story line of what the purpose is of the tool

To test: no new code was added, inspecting the README differences should be enough

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +31,50 @@ Programmatically generate synthetic sequence data such as time series, strings,
...
+parameter_space_0 = ParameterSpace([
+    Dimension("location", 8, 12),
+    Dimension("stddev", 1, 2),
+    Dimension("height", 4, 5),
+])
+
+channels = []
+channels.append(normal_peak(time_axis, **parameter_space_0.sample()))
+channels.append(gaussian(time_axis))
+channels.append(channels[0] + channels[1])
```
Comment (sverhoeven) [inline README.md:64]: Looks a bit clunky,

**Review by sverhoeven — CHANGES_REQUESTED** [review]
Can you move the png to `docs/_static` so we dont pollute the root directory.

File: README.md (hunk 1 above)
Comment (sverhoeven) [reply README.md]: A lot nicer now

**Review by sverhoeven — APPROVED** [review]
Nice to have an example in the README

**Review by sverhoeven — APPROVED** [review]
Thanks now with plot from docs/_static

**Comment by eriktks** [comment]
Thanks!

**Pull request merged**